util/mapcache_seed.c

```diff
@@ -262,6 +263,9 @@ static const apr_getopt_option_t seed_options[] = {
 #endif
   { "transfer", 'x', TRUE, "tileset to transfer" },
   { "zoom", 'z', TRUE, "min and max zoomlevels to seed, separated by a comma. eg 0,6" },
+#ifdef USE_CLIPPERS
+  { "ogr-not-touches", 'T', FALSE, "skip tiles only touching features"},
```

Comment:
I'd prefer it if this option did not have a short comandline argument as it would be rarely used. see e.g. https://github.com/tbonfort/mapcache/blob/oneatlas/util/mapcache_seed.c#L271